From 18d4c1b0ea1da0927b876602ae82cac6da93ddd2 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 6 May 2019 10:27:09 +0530
Subject: [PATCH 3/4] Don't error out if get_git_info() fails

---
 lib/cpy_distutils.py | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/lib/cpy_distutils.py b/lib/cpy_distutils.py
index 16af365..0b7dc8a 100644
--- a/lib/cpy_distutils.py
+++ b/lib/cpy_distutils.py
@@ -688,7 +688,12 @@ class BuildExtDynamic(build_ext):
     def run(self):
         """Run the command"""
         # Generate docs/INFO_SRC
-        git_info = get_git_info()
+        git_info = None
+        try:
+            git_info = get_git_info()
+        except Exception:
+            pass
+
         if git_info:
             with open(os.path.join("docs", "INFO_SRC"), "w") as info_src:
                 info_src.write("version: {}\n".format(VERSION_TEXT))
-- 
2.26.2

