From 62517589520341797834f4ecb653b6dc27e0e5dd Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Tue, 20 Oct 2020 11:39:43 -0400
Subject: [PATCH 4/4] Honor extra_compile_args for _mysqlxpb extension too

---
 cpydist/__init__.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/cpydist/__init__.py b/cpydist/__init__.py
index 4592dda..85da69a 100644
--- a/cpydist/__init__.py
+++ b/cpydist/__init__.py
@@ -504,6 +504,10 @@ class BuildExt(build_ext, BaseCommand):
                     "libprotobuf" if os.name == "nt" else "protobuf")
                 # Add -std=c++11 needed for Protobuf 3.6.1
                 ext.extra_compile_args.append("-std=c++11")
+                # Add extra compile args
+                if self.extra_compile_args:
+                    ext.extra_compile_args.extend(
+                        self.extra_compile_args.split())
                 self._run_protoc()
             if ext.name == "_mysql_connector":
                 if not self.with_mysql_capi:
-- 
2.26.2

