From 4c061cede5d77f66fd0c626d842020eecd5efaf7 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Tue, 20 Oct 2020 13:16:22 -0400
Subject: [PATCH 04/10] Fix indentation for common flags

---
 cpydist/__init__.py | 46 ++++++++++++++++++++++-----------------------
 1 file changed, 23 insertions(+), 23 deletions(-)

diff --git a/cpydist/__init__.py b/cpydist/__init__.py
index b445944..67094f4 100644
--- a/cpydist/__init__.py
+++ b/cpydist/__init__.py
@@ -749,29 +749,29 @@ class BuildExt(build_ext, BaseCommand):
                     )
                     disabled.append(ext)
                     continue
-                # Add extra compile args
-                if self.extra_compile_args:
-                    ext.extra_compile_args.extend(self.extra_compile_args.split())
-                # Add extra link args
-                if self.extra_link_args:
-                    ext.extra_link_args.extend(self.extra_link_args.split())
-                # Add -rpath if the platform is Linux
-                if platform.system() == "Linux" and not self.skip_vendor:
-                    ext.extra_link_args.extend(["-Wl,-rpath,$ORIGIN/mysql/vendor"])
-                # Add include dirs
-                if self.with_openssl_include_dir:
-                    ext.include_dirs.append(self.with_openssl_include_dir)
-                if "include_dirs" in self._mysql_info:
-                    ext.include_dirs.extend(self._mysql_info["include_dirs"])
-                # Add library dirs
-                ext.library_dirs.append(self._build_mysql_lib_dir)
-                if "library_dirs" in self._mysql_info:
-                    ext.library_dirs.extend(self._mysql_info["library_dirs"])
-                if self.with_openssl_lib_dir:
-                    ext.library_dirs.append(self.with_openssl_lib_dir)
-                # Add libraries
-                if "libraries" in self._mysql_info:
-                    ext.libraries.extend(self._mysql_info["libraries"])
+            # Add extra compile args
+            if self.extra_compile_args:
+                ext.extra_compile_args.extend(self.extra_compile_args.split())
+            # Add extra link args
+            if self.extra_link_args:
+                ext.extra_link_args.extend(self.extra_link_args.split())
+            # Add -rpath if the platform is Linux
+            if platform.system() == "Linux" and not self.skip_vendor:
+                ext.extra_link_args.extend(["-Wl,-rpath,$ORIGIN/mysql/vendor"])
+            # Add include dirs
+            if self.with_openssl_include_dir:
+                ext.include_dirs.append(self.with_openssl_include_dir)
+            if "include_dirs" in self._mysql_info:
+                ext.include_dirs.extend(self._mysql_info["include_dirs"])
+            # Add library dirs
+            ext.library_dirs.append(self._build_mysql_lib_dir)
+            if "library_dirs" in self._mysql_info:
+                ext.library_dirs.extend(self._mysql_info["library_dirs"])
+            if self.with_openssl_lib_dir:
+                ext.library_dirs.append(self.with_openssl_lib_dir)
+            # Add libraries
+            if "libraries" in self._mysql_info:
+                ext.libraries.extend(self._mysql_info["libraries"])
             # Suppress unknown pragmas
             if os.name == "posix":
                 ext.extra_compile_args.append("-Wno-unknown-pragmas")
