From 9d146ec7c3acc6cfb50fcf26d0907509ddbafdca Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sat, 10 Jun 2023 09:49:29 +0100
Subject: [PATCH 07/10] Delete openssl (vendor-ed) libs only if provided

---
 cpydist/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpydist/__init__.py b/cpydist/__init__.py
index 59f28f1..8bb6089 100644
--- a/cpydist/__init__.py
+++ b/cpydist/__init__.py
@@ -264,7 +264,7 @@ class BaseCommand(Command):
             vendor_folder = os.path.join(os.getcwd(), self.vendor_folder)
             if os.path.exists(vendor_folder):
                 shutil.rmtree(vendor_folder)
-            elif os.name == "nt":
+            elif os.name == "nt" and self.with_openssl_lib_dir:
                 libssl, libcrypto = self._get_openssl_libs(ext="dll")
                 libraries = ["libmysql.dll", libssl, libcrypto]
                 for filename in libraries:
