From 7186c3e7aa3682efd957e367ef2a0bacb07301cc Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Wed, 21 Feb 2024 20:27:00 +0100
Subject: [PATCH 10/10] Link to abseil_dll on Windows

---
 cpydist/__init__.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cpydist/__init__.py b/cpydist/__init__.py
index d62a794..f0d290b 100644
--- a/cpydist/__init__.py
+++ b/cpydist/__init__.py
@@ -738,6 +738,7 @@ class BuildExt(build_ext, BaseCommand):
                 ext.libraries.append("libprotobuf" if os.name == "nt" else "protobuf")
                 # Add -std=c++11 needed for Protobuf 3.6.1
                 if os.name == "nt":
+                    ext.libraries.append("abseil_dll")
                     ext.extra_compile_args.append("/std:c++17")
                 else:
                     ext.extra_compile_args.append("-std=c++17")
